### ISSUE TITLE
New draft.yaml syntax - config

### ISSUE THREAD

**Issue by benjam-es** [laravel-shift/blueprint#529]
### Synopsis:
As a further improvement to the configurability of the package, we could simplify things for those who want to make single simple changes to the configuration often, without needing to publish the full config file and make an edit, then re-run the build command

### Proposed Syntax:
```yaml
models:
  ModelOne:
    column: datatype modifier

  ModelTwo:
    column: datatype
    another_column: datatype modifier

config:
    namespace: MyAppNamespace
    generatePhpdocs: true

````

### Expected Behavior:
Streamline the process for users who often just need simple tweaks. Also provides a visual reference of changes users want with a specific draft file.

This would read the tree for config, and merge the config overrides from the draft yaml file

**Comment by jasonmccreary** [comment]
Having a superseding way to overwrite config in the Draft file could be helpful for those one-offs. Much like config in a `phpunit.xml` file. However, we shouldn't replace the config file entirely. Just allow temporarily adjusting it for a build.

**Comment by jasonmccreary** [comment]
Merged in #575.
